---a/net/minecraft/nbt/NBTTagInt.java
+++b/net/minecraft/nbt/NBTTagInt.java
@@ -4,90 +4,74 @@
 import java.io.DataOutput;
 import java.io.IOException;
 
-public class NBTTagInt extends NBTBase.NBTPrimitive
-{
+public class NBTTagInt extends NBTBase.NBTPrimitive {
+
     private int data;
     private static final String __OBFID = "CL_00001223";
 
-    NBTTagInt() {}
+    NBTTagInt() {
+    }
 
-    public NBTTagInt(int data)
-    {
+    public NBTTagInt(int data) {
         this.data = data;
     }
 
-    void write(DataOutput output) throws IOException
-    {
+    void write(DataOutput output) throws IOException {
         output.writeInt(this.data);
     }
 
-    void read(DataInput input, int depth, NBTSizeTracker sizeTracker) throws IOException
-    {
+    void read(DataInput input, int depth, NBTSizeTracker sizeTracker) throws IOException {
         sizeTracker.read(32L);
         this.data = input.readInt();
     }
 
-    public byte getId()
-    {
-        return (byte)3;
+    public byte getId() {
+        return (byte) 3;
     }
 
-    public String toString()
-    {
+    public String toString() {
         return "" + this.data;
     }
 
-    public NBTBase copy()
-    {
+    public NBTBase copy() {
         return new NBTTagInt(this.data);
     }
 
-    public boolean equals(Object p_equals_1_)
-    {
-        if (super.equals(p_equals_1_))
-        {
-            NBTTagInt nbttagint = (NBTTagInt)p_equals_1_;
+    public boolean equals(Object p_equals_1_) {
+        if (super.equals(p_equals_1_)) {
+            NBTTagInt nbttagint = (NBTTagInt) p_equals_1_;
             return this.data == nbttagint.data;
-        }
-        else
-        {
+        } else {
             return false;
         }
     }
 
-    public int hashCode()
-    {
+    public int hashCode() {
         return super.hashCode() ^ this.data;
     }
 
-    public long getLong()
-    {
-        return (long)this.data;
+    public long getLong() {
+        return (long) this.data;
     }
 
-    public int getInt()
-    {
+    public int getInt() {
         return this.data;
     }
 
-    public short getShort()
-    {
-        return (short)(this.data & 65535);
+    public short getShort() {
+        return (short) (this.data & 65535);
     }
 
-    public byte getByte()
-    {
-        return (byte)(this.data & 255);
+    public byte getByte() {
+        return (byte) (this.data & 255);
     }
 
-    public double getDouble()
-    {
-        return (double)this.data;
+    public double getDouble() {
+        return (double) this.data;
     }
 
-    public float getFloat()
-    {
-        return (float)this.data;
+    public float getFloat() {
+        return (float) this.data;
     }
-}
+}
